pkg/cluster/k8sres.go

```diff
@@ -941,6 +941,11 @@ func (c *Cluster) getPodEnvironmentSecretVariables() ([]v1.EnvVar, error) {
 	return secretPodEnvVarsList, nil
 }
 
+func (c *Cluster) podEnvironmentObjectNameTemplate(template string) string {
+	objectNameTemplate := config.StringTemplate(template)
+	return objectNameTemplate.Format("cluster", c.clusterName().Name, "tprkind", acidv1.PostgresCRDResourceKind, "tprgroup", acidzalando.GroupName)
```

Comment:
why should the template contain the CRD kind and group?

Reply:
I got the same template as operator has for the secret names to be unique name across different types and be a separating. Probably, this options can be omitted if not needed. Also, we could probably add a `CONTROLLER_ID` to have unique name across clusters and controllers.